resources/default_contacts.json

```diff
@@ -36,7 +36,7 @@
         {
             "en": "http://auctionhouse.dappbench.com"
         }
-    },
+   },
```

Comment:
ok, I've fixed the tabulation error. The commits can be squashed when the pull request is merged, correct?